lib/afinter.h

```diff
@@ -28,6 +28,13 @@
 #include "driver.h"
 #include "logsource.h"
 
+enum
```

Comment:
You should assign a name to this enum to reference it later as a type.
I will mark later where it should be used: (*)

Reply:
@jszigetvari : Okay